src/java/org/apache/cassandra/db/virtual/AccordVirtualTables.java

```diff
@@ -18,57 +18,205 @@
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
 
 public class AccordVirtualTables
 {
-    private AccordVirtualTables()
-    {
-
-    }
+    private AccordVirtualTables() {}
 
     public static Collection<VirtualTable> getAll(String keyspace)
     {
         if (!DatabaseDescriptor.getAccordTransactionsEnabled())
             return Collections.emptyList();
 
-        return Arrays.asList(
-        new Epoch(keyspace)
+        return List.of(
+            new CommandStoreCache(keyspace),
+            new MigrationStates(keyspace),
+            new CoordinationStatus(keyspace)
         );
     }
 
-    @VisibleForTesting
-    public static final class Epoch extends AbstractVirtualTable
+    public static final class CommandStoreCache extends AbstractVirtualTable
     {
+        private CommandStoreCache(String keyspace)
+        {
+            super(parse(keyspace,
+                        "Accord Command Store Cache Metrics",
+                        "CREATE TABLE accord_command_store_cache(\n" +
+                        "  id int,\n" +
+                        "  scope text,\n" +
+                        "  queries bigint,\n" +
+                        "  hits bigint,\n" +
+                        "  misses bigint,\n" +
+                        "  PRIMARY KEY (id, scope)" +
+                        ')'));
+        }
 
-        protected Epoch(String keyspace)
+        @Override
+        public DataSet data()
         {
-            super(parse(keyspace, "Accord Epochs",
-                        "CREATE TABLE accord_epochs(\n" +
-                        "  epoch bigint,\n" +
-                        "  PRIMARY KEY ( (epoch) )" +
-                        ")"));
+            CommandStores stores = ((AccordService) AccordService.instance()).node().commandStores();
+            SimpleDataSet result = new SimpleDataSet(metadata());
+
+            List<AccordStateCache.ImmutableStats> commandStats =
+                AsyncChains.getBlockingAndRethrow(stores.map(store -> ((AccordCommandStore) store.commandStore()).commandCache().statsSnapshot()));
+            for (int storeID : stores.ids())
+                addRow(commandStats.get(storeID), result, storeID, "command");
+
+            List<AccordStateCache.ImmutableStats> commandsForKeyStats =
+                AsyncChains.getBlockingAndRethrow(stores.map(store -> ((AccordCommandStore) store.commandStore()).commandsForKeyCache().statsSnapshot()));
+            for (int storeID : stores.ids())
+                addRow(commandsForKeyStats.get(storeID), result, storeID, "commands_for_key");
+
+            List<AccordStateCache.ImmutableStats> timestampsForKeyStats =
+                AsyncChains.getBlockingAndRethrow(stores.map(store -> ((AccordCommandStore) store.commandStore()).timestampsForKeyCache().statsSnapshot()));
+            for (int storeID : stores.ids())
+                addRow(timestampsForKeyStats.get(storeID), result, storeID, "timestamps_for_key");
```

Comment:
nit: why schedule on the same stores 3 times when you could do this once?  this would be 3x faster to do